rocAL/rocAL/source/pipeline/ring_buffer.cpp

```diff
@@ -142,15 +134,15 @@ void RingBuffer::init(RocalMemType mem_type, void *devres, unsigned sub_buffer_s
         {
             cl_mem_flags flags = CL_MEM_READ_ONLY;
 
-            _dev_sub_buffer[buffIdx].resize(_sub_buffer_count);
-            for(unsigned sub_idx = 0; sub_idx < _sub_buffer_count; sub_idx++)
+            _dev_sub_buffer[buffIdx].resize(sub_buffer_count);
```

Comment:
Please align the code under the for loop at L137

Reply:
I think it is aligned please lmk if it is wrong